### PR TITLE
[Setup] Update SwiftLint shell script and fix linter warnings

### PR DESCRIPTION
**Pull request by yo1995**
## Description

This PR updates the build phase. I'm using SwiftLint 0.51.0 to fix warnings.

See https://github.com/realm/SwiftLint#xcode

### CODE REVIEW

File: Shared/Samples/Sketch on map/SketchOnMapView.swift [hunk 1]
```diff
@@ -183,7 +183,8 @@ extension GeometryEditorMenu {
 }
 
 /// An object that acts as a view model for the geometry editor menu.
-@MainActor class GeometryEditorMenuModel: ObservableObject {
+@MainActor
+class GeometryEditorMenuModel: ObservableObject {
```
Comment (dfeinzimer) [inline Shared/Samples/Sketch on map/SketchOnMapView.swift:187]: Were you getting warnings on these? I'm not if I undo the changes.

**Review by dfeinzimer — APPROVED** [review]

File: Shared/Samples/Sketch on map/SketchOnMapView.swift (hunk 1 above)
Comment (yo1995) [reply Shared/Samples/Sketch on map/SketchOnMapView.swift:187]: Are you using SwiftLint 0.51.0?

Comment (dfeinzimer) [reply Shared/Samples/Sketch on map/SketchOnMapView.swift:187]: I wasn't before but I updated and I still am not. It's okay though, no need to hold the PR for this.

Comment (dfeinzimer) [reply Shared/Samples/Sketch on map/SketchOnMapView.swift:187]: Scratch that, I just built again and the warnings finally came up 😌

Comment (yo1995) [reply Shared/Samples/Sketch on map/SketchOnMapView.swift:187]: image

Hmm interesting. I don't know why these warnings popped up after I upgraded. The rule seems to be pretty old and I cannot find a related PR (might be https://github.com/realm/SwiftLint/pull/4616 ?)

I'll merge this PR as it is not giving warnings on your end.

Comment (yo1995) [reply Shared/Samples/Sketch on map/SketchOnMapView.swift:187]: Hehe thanks for the update. 😬
